db/migrate/20240709190021_add_categories_to_rating.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class AddCategoriesToRating < ActiveRecord::Migration[7.1]
+  def up
+    add_column :ratings, :originality, :integer, null: false, default: 1
+    add_column :ratings, :description_quality, :integer, null: false, default: 1
+    add_column :ratings, :test_quality, :integer, null: false, default: 1
+    add_column :ratings, :model_solution_quality, :integer, null: false, default: 1
+
+    ActiveRecord::Base.connection.execute(
+      %(UPDATE ratings SET
+                originality = rating,
+                description_quality = rating,
+                test_quality = rating,
+                model_solution_quality = rating)
+    )
```

Comment:
Feel free to use the following:

```suggestion
execute <<~SQL.squish
  UPDATE ratings
  SET originality = rating,
      description_quality = rating,
      test_quality = rating,
      model_solution_quality = rating
SQL
```